### ISSUE TITLE
Additional Information on Solar System

### ISSUE THREAD

**Issue by pascalforny** [OpenExoplanetCatalogue/open_exoplanet_catalogue#804]
Are there any plans to integrate more data about bodies in our own solar system such as roatation period or axis tilt?

**Comment by hannorein** [comment]
No. There is some of that information in the oec_outreach repository. Have a look, maybe that's what you're interested in..
